.travis.yml

```diff
@@ -19,5 +21,7 @@ before_install:
   - sudo DEBIAN_FRONTEND=noninteractive apt-get install -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" docker-engine
 
 script:
-    - cd linux/test && ./docker-build.sh $ENV && docker run -d omero_install_test_$ENV
+    - cd linux/test
+    - ./docker-build.sh $ENV
+    - if [[ "$ENV" == "centos7" ]]; then ./test_services.sh; fi
```

Comment:
`test_services.sh` looks quite nice, but is now only being run on a subset, right?